mongo/integration/unified/event_verification.go

```diff
@@ -150,5 +352,10 @@ func stringifyEventsForClient(client *clientEntity) string {
 		str.WriteString(fmt.Sprintf("[%s] CommandName: %s, Failure: %s\n", evt.ConnectionID, evt.CommandName, evt.Failure))
 	}
 
+	str.WriteString("\nPool Events\n\n")
+	for _, evt := range client.poolEvents() {
+		str.WriteString(fmt.Sprintf("[%s] Event Type: %q\n", evt.Address, evt.Type))
```

Comment:
[super-nit] why not `Event Type: %T\n`?

Reply:
The `%T` directive would print out the Go language type of the variable. The `evt.Type` field is a string, so that would just print "string". The `%q` directive will actually print out the value stored in the `Type` field. See https://play.golang.org/p/xA4nSgXOxMC for an example.

Reply:
Ah gotcha. Didn't realize `Type` was a string haha.